waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

```diff
@@ -277,13 +277,13 @@ public void typicalWithGraphite() throws Exception {
     assertMetric(metrics,
         "graphitePrefix.counter.com.hotels.bdp.waggledance.server.FederatedHMSHandler.get_all_databases.all.success.count 2");
     assertMetric(metrics,
-        "graphitePrefix.counter.com.hotels.bdp.waggledance.server.FederatedHMSHandler.get_table.primary.calls.count 1");
+        "graphitePrefix.counter.com.hotels.bdp.waggledance.server.FederatedHMSHandler.get_table_req.primary.calls.count 1");
```

Comment:
Hmmm, so does this mean that getTable() at the client level is being translated into something new (get_table_req) instead of get_table at the Thrift layer? What happened to the get_table code path? How different are they? Is this the part we're worried might not be backwards compatible?

Reply:
I guess if an 2.1 client fetches a table then get_table will be invoked. It should be fine as long as both the actual metastore and WD use the same Hive version. The problem is if the real metastore runs 2.1, WD 2.3 and the client 2.3. I'll be running functional tests next week so we can decided wether we start maintaining a development branch for each of these versions and possibly future versions

Reply:
OK, we discussed this offline, looks like if things are running the same version it's all fine and the backwards compatible use cases that we care about work too so this is fine.